src/middlewared/middlewared/plugins/network.py

```diff
@@ -1091,10 +1093,18 @@ def _get_filters(key):
             return [[key, '!=', update['id']]] if update else []
 
         validation_attrs = {
-            'aliases': ['Active node IP address', ' cannot be changed.', ' is required when configuring HA'],
-            'failover_aliases': ['Standby node IP address', ' cannot be changed.', ' is required when configuring HA'],
-            'failover_virtual_aliases': ['Virtual IP address', ' cannot be changed.', ' is required when configuring HA'],
-            'failover_group': ['Failover group number', ' cannot be changed.' ' is required when configuring HA'],
+            'aliases': [
```

Comment:
Can you please clarify why we have changed this ?

Reply:
flake8 fix one of the lines were too long but I changed all of them so they were all consistent.

Reply:
We have merged this, but i think we should have flake8 changes in a different PR to limit the scope of a PR.